src/components/__tests__/Discipline-test.js

```diff
@@ -0,0 +1,16 @@
+import 'react-native'
+import React from 'react'
+import renderer from 'react-test-renderer'
+import Discipline from '../Discipline'
+
+it('renders with a title', () => {
+  const tree = renderer.create(
+    <Discipline
+      title='O Hai!'
+      icon='arts'
+      tag='test'
+      color='#fff'
+      setSelectedProjectTag={jest.fn} />
+  ).toJSON()
+  expect(tree).toMatchSnapshot()
```

Comment:
Yes, and then in the future if the rendered output is different, it will cause the test to fail (meaning it needs to be re-generated)